sim-cli/src/main.rs

```diff
@@ -15,34 +14,51 @@ use simple_logger::SimpleLogger;
 #[derive(Parser)]
 #[command(version)]
 struct Cli {
-    #[clap(index = 1)]
-    sim_file: PathBuf,
+    /// Path to folder containing simulation files and where simulation results will be written
+    #[arg(index = 1, env = "SIMLN_DATA_DIR")]
```

Comment:
Breaking change to change the first argument from `sim-file` to needing to provide a data dir. 

How about assuming that the working directory is the data dir and having this as a flag to point somewhere else if desired?